<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bussines Model Optimization</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>420</width>
     <height>271</height>
    </size>
   </property>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>411</width>
      <height>261</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuModelo">
    <property name="title">
     <string>Modelo</string>
    </property>
    <addaction name="actionEditar_Modelo"/>
    <addaction name="actionAgregar_elemento"/>
    <addaction name="actionOptimizar"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionAyuda"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuModelo"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionAgregar_elemento"/>
   <addaction name="actionOptimizar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionAgregar_elemento">
   <property name="text">
    <string>Agregar elemento</string>
   </property>
  </action>
  <action name="actionOptimizar">
   <property name="text">
    <string>Optimizar</string>
   </property>
  </action>
  <action name="actionEditar_Modelo">
   <property name="text">
    <string>Editar Modelo</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
